test/clj_meowallet/core_test.clj

```diff
@@ -73,3 +74,22 @@
 
   (testing "success"
     (is (result/failed? result)))))
+
+(deftest verify-callback-test
+  (if-not (env :meo-wallet-api-key)
+    (println "Warning: No meo wallet api key on env (ignoring test)")
+
+    (let [credentials {:meo-wallet-api-key (env :meo-wallet-api-key)}
+          data {:amount 10
+                :currency "EUR"
+                :event "COMPLETED"
+                :ext-customerid "00001"
+                :ext-email "noreply@sapo.pt"
+                :ext-invoiceid "38440200100"
+                :method "WALLET"
+                :operation-id "qwkjehqkjwhe"
+                :operation-status "COMPLETED"
+                :user "237"}
+        result (<!! (core/verify-callback credentials data))]
```

Comment:
What I mean is that that will always fail because it's not correct auth for that request. So if you just want to cover code, you could check for a specific return. Or else this could fail because the endpoint changed and the test will still pass.